test/conformance/api/v1/util.go

```diff
@@ -49,12 +55,25 @@ func checkForExpectedResponses(ctx context.Context, t testing.TB, clients *test.
 }
 
 func validateDomains(t testing.TB, clients *test.Clients, baseDomain *url.URL,
```

Comment:
I think you can drop baseDomain as well since we're fetching the service which would have it.